js/client/modules/@arangodb/testsuites/arangosh.js

```diff
@@ -185,6 +186,13 @@ function arangosh (options) {
     tmpMgr.destructor(ret[section]['status'] && options.cleanup);
   }
 
+  runTest('testArangoshExitVersion',
+          'Starting arangosh printing the version:',
+          '',
+          0,
+          {'version': true});
```

Comment:
Probably works as is, but IMO all command-line arguments will end up as strings anyway.
```suggestion
          {'version': 'true'});
```